main.go

```diff
@@ -30,7 +34,36 @@ func setupFlags() {
 	flag.Parse()
 }
 
+func getLogLevel() slog.Leveler {
```

Comment:
not entirely happy with this being here, what would be the right place for it?

Reply:
It can be moved to a separate package.